src/View/Requirements_Backend.php

```diff
@@ -1421,6 +1439,39 @@ function () use ($fileList, $minify, $type) {
         return $combinedURL;
     }
 
+    /**
+     * Resolves relative paths in CSS files which are lost when combining them
+     *
+     * @param string $content
+     * @param string $filePath
+     * @return string New content with paths resolved
+     */
+    protected function resolveCSSReferences($content, $filePath)
+    {
+        $doResolving = Config::inst()->get(__CLASS__, 'resolve_relative_css_refs');
+        if (!$doResolving) {
+            return $content;
+        }
+        $fileUrl = Injector::inst()->get(ResourceURLGenerator::class)->urlForResource($filePath);
+        $fileUrlDir = dirname($fileUrl);
+        $content = preg_replace_callback('#([\s\'"\(])((:?[\.]{1,2}/)+)#', function ($a) use ($fileUrlDir) {
+            [ $_, $prefix, $match ] = $a;
+            $full = $fileUrlDir . '/' . $match;
+            $full = preg_replace('#/{2,}#', '/', $full); // ensure there's no repeated slashes
```

Comment:
What is this trying to resolve? If two slashes in a row is actually invalid, then we should leave this alone (it'll produce the same issues as with the non-combined file and the developer can then resolve those issues)

Reply:
It's just a courtesy action... actually Symfony's Path routines do the same during canonicalization. I'd prefer to leave it as is.

Reply:
I'm fine with it happening as a side effect of `Path` methods.